src/components/AppLayout/AppChannelContext.tsx

```diff
@@ -37,11 +37,18 @@ const isValidChannel = (
   return channelList?.some(getById(channelId));
 };
 
-export const AppChannelProvider: React.FC = ({ children }) => {
-  const { isAuthenticated } = useAuth();
+interface AppChannelProviderProps {
+  onChannelChange: (channel: string) => void;
+}
+
+export const AppChannelProvider: React.FC<AppChannelProviderProps> = ({
+  children,
+  onChannelChange
+}) => {
+  const { authenticated, authenticating } = useAuth();
   const [selectedChannel, setSelectedChannel] = useLocalStorage("channel", "");
   const { data: channelData, refetch } = useBaseChannelsList({
-    skip: !isAuthenticated
+    skip: !authenticated || authenticating
```

Comment:
```suggestion
    skip: !authenticated
```
We only care if the user is `authenticated`, why do we need to check if it's in flight as well?